### ISSUE TITLE
[FEAT] Set Default Database

### ISSUE THREAD

**Issue by sahriVoraus** [PhilippC/keepass2android#2582]
It would be nice to have the possibility to set one of several databases as default. Maybe even add the possibility to always select the database on startup when all are locked.

Issue: I have a separate database for work that I use on my mobile only once in a while. Mostly I open it additionally to my private one with "Open another database". But when my databases are locked after awhile, keepass2android will always start with the work database. The steps to switch back to my regular database then - starting with canceling the finger print scan, the appearing keyboard (cause the password field is automatically selected),... - is a little cumbersome.

Maybe it would already help to not start with the database that was just opened via "Open another database".
